src/compose/application-manager.ts

```diff
@@ -55,19 +53,9 @@ export const removeAllListeners: typeof events['removeAllListeners'] = events.re
 	events,
 );
 
-const proxyvisor = new Proxyvisor();
+export const proxyvisor = new Proxyvisor();
```

Comment:
I don't like this, and the fact that the API has to import application manager.  What if we modify the `proxyvisor` module to have a default export ? Then both application manager and API can import the proxyvisor

```
// proxyvisor.ts

const proxyvisor = new Proxyvisor();
export default proxyvisor
```

Reply:
Done with https://github.com/balena-os/balena-supervisor/commit/5714830fceef45714312d08aeb5229b8324eb2c7